folly/Singleton.h

```diff
@@ -200,7 +200,11 @@ class SingletonHolderBase {
   virtual void destroyInstance() = 0;
 
  protected:
+#ifdef MSVC_NO_STATIC_INCLASS_CONSTEXPR_INITIALIZATION
+  static const std::chrono::seconds kDestroyWaitTime;
+#else
   static constexpr std::chrono::seconds kDestroyWaitTime{5};
+#endif
```

Comment:
If we're supporting broken compilers, just make this `const` rather than `constexpr` for all compilers. And add a comment explaining why.

Or keep this as `constexpr` but move it out of the class and into the namespace, renaming it to

`kSingletonHolderBaseDestroyWaitTime`

And add a comment why it can't go into the class.